community/libfts/build

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+./bootstrap.sh
+./configure \
+    --prefix=/usr
+
+make
+make \
+    DESTDIR="$1" \
+    install
```

Comment:
`make \DESTDIR="$1" \install` can be on a single line (optionally forgoing `DESTDIR="$1"`).